lib/teiserver_web/live/account/profile/contributor.ex

```diff
@@ -69,19 +71,27 @@ defmodule TeiserverWeb.Account.ProfileLive.Contributor do
 
     {:noreply,
      socket
-     |> assign(:temp_country_code, "")}
+     |> assign(:temp_country_code, "")
+     |> assign(:error_message, nil)}
   end
 
   def handle_event("save-country_code", _, %{assigns: assigns} = socket) do
-    Account.update_user_stat(assigns.user.id, %{
-      "bar_plus.flag" => assigns.temp_country_code
-    })
-
-    Account.recache_user(assigns.user.id)
-
-    {:noreply,
-     socket
-     |> assign(:country_code, assigns.temp_country_code)}
+    if String.match?(assigns.temp_country_code, ~r/^[a-zA-Z0-9-]{2,8}$/) do
```

Comment:
Note to self: verify that this passes with all the flags in the chobby repo.

Reply:
One does not:
`europeanunion.png`

Reply:
Please allow up to 16 characters in the regex, to be sure

Reply:
`eu.png` already exists for EU, `europeanunion` is a duplicate.

Reply:
Increased the size anyway